client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -138,7 +138,7 @@ class GoogleAnalyticsForm extends Component {
 			: translate( 'Enable Google Analytics by upgrading to the Business plan' );
 
 		return (
-			<form id="site-settings" onSubmit={ handleSubmitForm }>
```

Comment:
Are we sure this wasn't used before, e.g. for styling?

Reply:
Yes, I verified that. We're not using `#site-settings` anywhere. We're using `.site-settings` quite a bit though:

https://github.com/Automattic/wp-calypso/blob/cb1c1d8b5d7691e077285c462a8e7aaa61011778/client/my-sites/site-settings/style.scss#L1